ios/FluentUI/Tooltip/TooltipView.swift

```diff
@@ -0,0 +1,425 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+// MARK: TooltipView
+
+class TooltipView: UIView {
+
+    init(anchorView: UIView,
+         message: String,
+         title: String? = nil,
+         textAlignment: NSTextAlignment,
+         preferredArrowDirection: Tooltip.ArrowDirection,
+         offset: CGPoint,
+         arrowMargin: CGFloat,
+         tokenSet: TooltipTokenSet) {
+        self.anchorView = anchorView
+        self.message = message
+        self.titleMessage = title
+        self.preferredArrowDirection = preferredArrowDirection
+        self.offset = offset
+        self.arrowMargin = arrowMargin
+        self.tokenSet = tokenSet
+
+        let arrowImageViewBaseImage = UIImage.staticImageNamed("tooltip-arrow")
+        arrowImageView = UIImageView(image: arrowImageViewBaseImage)
+        arrowImageView.image = arrowImageViewBaseImage?.withTintColor(UIColor(dynamicColor: tokenSet[.tooltipColor].dynamicColor), renderingMode: .alwaysOriginal)
+
+        super.init(frame: .zero)
+
+        self.addSubview(backgroundView)
+        self.accessibilityViewIsModal = true
+
+        arrowImageView.transform = transformForArrowImageView()
+        self.addSubview(arrowImageView)
+
+        messageLabel.text = message
+        messageLabel.textAlignment = textAlignment
+
+        if let titleLabel = titleLabel, let title = title {
+            titleLabel.text = title
+            titleLabel.textAlignment = textAlignment
+        }
+
+        self.addSubview(textContainer)
+
+        // Shadow
```

Comment:
can we add a todo not to integrate  with https://github.com/microsoft/fluentui-apple/pull/1296 ? @laminesm we should track this separately as well